internal/pkg/docker/dockerfile/dockerfile_test.go

```diff
@@ -196,3 +195,66 @@ EXPOSE 8080/tcp 5000`),
 		})
 	}
 }
+
+func TestParseHealthCheckDockerfile(t *testing.T) {
```

Comment:
I believe for methods, the test names should be: `TestDockerfile_GetHealthCheck`